Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYPaymentProvider.h

```diff
@@ -61,6 +62,14 @@ extern NSString *const BUYPaymentProviderDidCompleteCheckoutNotificationKey;
 @optional
 
 /**
+ *  Called when a controller needs to be presented
```

Comment:
should include more details here.  Why is this optional, what should be called if its implemented.